tests/aws/services/events/test_events.py

```diff
@@ -706,18 +706,113 @@ def _get_sqs_messages():
         assert message_body["time"] == "2022-01-01T00:00:00Z"
 
 
-class TestEventsEventBus:
+class TestEventBus:
     @markers.aws.validated
-    def test_create_custom_event_bus(self, aws_client, cleanups, snapshot):
+    @pytest.mark.skipif(
+        not is_v2_provider() and not is_aws_cloud(),
+        reason="V1 provider does not support this feature",
+    )
+    @pytest.mark.parametrize("regions", [["us-east-1"], ["us-east-1", "us-west-1", "eu-central-1"]])
+    def test_create_list_describe_delete_custom_event_buses(
```

Comment:
These tests seem flaky and we should fix them: https://app.circleci.com/pipelines/github/localstack/localstack/24319/workflows/83b9939c-a5e2-4b19-becd-2c40030242d4/jobs/201263/tests

That's related to the [previous concurrency comment](https://github.com/localstack/localstack/pull/10613/files#r1567410818), because snapshotting lists can be flaky when other tests don't clean up properly (as you guessed Max).

We can xFail them to move on but should fix them ultimately.

Reply:
v2 tests all clean up correctly so far, and this is a core test for the v2 functionality this is why I would strongly prefer to keep it in for the v2 provider, non the less it is already a part of the refactor tests backlog.

Reply:
as discussed, we cannot reasonably assume that all current and future tests perform perfect cleanup, and therefore we should take pre-cautions such that these tests don't flake.